core/storage/skill/gae_models.py

```diff
@@ -58,6 +58,8 @@ class SkillModel(base_models.VersionedModel):
         required=True, indexed=True)
     # A dict representing the skill contents.
     skill_contents = ndb.JsonProperty(indexed=False)
+    # The id to be used by the next misconception added.
+    next_misconception_id = ndb.StringProperty(required=True, indexed=False)
```

Comment:
As far as I can tell, this seems to be used as an int throughout. If that's the case why not just store it as an int?

Reply:
idk, it just seemed like ids in general were strings so I stuck with that, I can change it to int though

Reply:
changed to int